test/verify/check-networking-firewall

```diff
@@ -113,7 +113,7 @@ class TestFirewall(NetworkCase):
             self.wait_onoff("#firewall-heading-title-group", False)
             wait_unit_state(m, "firewalld", "active")
             # Wait until the default zone is listed
-            b.wait_in_text("#zones-listing .zone-section[data-id='public']", "public Zone")
+            b.wait_in_text("#zones-listing .zone-section[data-id='public']", "public zone")
```

Comment:
This probably needs -distropkg special casing. Probably better to use a constant for that and set it in setUp().